components/Snackbar/src/MDCSnackbarManager.m

```diff
@@ -241,11 +242,11 @@ - (void)displaySnackbarViewForMessage:(MDCSnackbarMessage *)message {
       showSnackbarView:snackbarView
               animated:YES
             completion:^{
-              if ([self isSnackbarTransient:snackbarView]) {
+              if (!snackbarView.accessibilityViewIsModal &&
```

Comment:
Tip: It would be nice for this conditional to match the one in the dismiss block below.  I would consider swapping the if/else blocks here so you can use the same set of logical conditions.

Reply:
Sure. Addressed in a5ddbbd01590490c2cebea9e302ad8fbdd774c28.